static/third/bootstrap/css/bootstrap.css

```diff
@@ -2497,7 +2497,7 @@ legend + .control-group {
 .nav > li > a:hover,
 .nav > li > a:focus {
   text-decoration: none;
-  background-color: #eeeeee;
+  background-color: hsla(0, 0%, 0%, 0.2);
```

Comment:
We don't make changes in bootstrap.css, since that's third-party code.  Also, is this rule only for the night theme?  If so, a change belongs in the `night-mode.scss` file.

Reply:
I have made the changes you can check it.